webpack.config.js

```diff
@@ -34,5 +34,8 @@ module.exports = {
         filename: 'index.html',
         pkg: pkg,
         template: path.join(testPath, 'index.html')
-    })]
+    })],
+    node: {
+        fs: "empty"
```

Comment:
First time I use webpack and I had some problem with it. When I run webpack .\webpack.config.dist.js, I have the next issue  :
Module not found: Error: Cannot resolve module 'fs'
Adding this line solved my problem.
But this was before than I realized that launching webpack command is not needed ^^' You have made some usefull scripts that work very well without this line. I will remove it :)